<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRunsPage</class>
 <widget class="QWidget" name="AddRunsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="RunSelectorWidget" name="run_selector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SummationSettingsWidget" name="summation_settings" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputFileGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="outputDirectoryLabel">
        <property name="text">
         <string>Output Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileNameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sumButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sum</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RunSelectorWidget</class>
   <extends>QWidget</extends>
   <header>ui.sans_isis.run_selector_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SummationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui.sans_isis.summation_settings_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
